components/org.wso2.carbon.identity.oauth.endpoint/src/main/java/org/wso2/carbon/identity/oauth/endpoint/util/EndpointUtil.java

```diff
@@ -811,6 +825,13 @@ public static String getUserConsentURL(OAuth2Parameters params, String loggedInU
                 // Append additional query params to the consent page url.
                 consentPageUrl = FrameworkUtils.appendQueryParamsStringToUrl(consentPageUrl, additionalQueryParams);
 
+                // Append scope metadata to the consent page url.
+                String metadataQueryParam = getScopeMetadataQueryParam(params.getConsentRequiredScopes(),
```

Comment:
Shall we change this variable name to `scopeMetaDataQueryParam`?  I think it will be better